ting_marc.module

```diff
@@ -431,8 +429,10 @@ function ting_marc_get_data($id, $field, $subfield) {
     return implode(' ', $result);
   }
 
-  // Default data loading.
-  $object = ting_marc_load_object($id);
+  // Default data loading: use core ting module's built-in funcionality to fetch
+  // marc objects from the data well.
+  module_load_include('client.inc', 'ting');
```

Comment:
We suggest to use `module_load_include('inc', 'ting', 'client');` instead of `module_load_include('client.inc', 'ting');` (https://api.drupal.org/api/drupal/includes%21module.inc/function/module_load_include/7.x).

@holt83 Will you change, or should we merge as is?

Reply:
@guddo Sure, I'll change it right away

Reply:
Done